typescript/.openapi-generator-ignore

```diff
@@ -21,3 +21,4 @@
 #docs/*.md
 # Then explicitly reverse the ignore rule for a single file:
 #!docs/README.md
+package.json
```

Comment:
do you also want to add `package-lock.json`?

Reply:
sure